### ISSUE TITLE
bug: Inconsistent result after apply: ip_access[*].description was null, but now cty.StringVal("")

### ISSUE THREAD

**Issue by tstraley** [ClickHouse/terraform-provider-clickhouse#28]
Per the docs, `ip_access[].description` should be optional, but when left null leads to bug and tainted resource:

eg.
```
  ip_access = [
    { source = var.cidr }
  ]
```

results in:

```
Acquiring state lock. This may take a few moments...
module.clickhouse.clickhouse_service.main: Creating...
module.clickhouse.clickhouse_service.main: Still creating... [10s elapsed]
module.clickhouse.clickhouse_service.main: Still creating... [20s elapsed]
module.clickhouse.clickhouse_service.main: Still creating... [30s elapsed]
module.clickhouse.clickhouse_service.main: Still creating... [40s elapsed]
module.clickhouse.clickhouse_service.main: Still creating... [50s elapsed]
╷
│ Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to module.clickhouse.clickhouse_service.main,
│ provider "provider[\"registry.terraform.io/clickhouse/clickhouse\"]"
│ produced an unexpected new value: .ip_access[0].description: was null, but
│ now cty.StringVal("").
│ 
│ This is a bug in the provider, which should be reported in the provider's
│ own issue tracker.
╵
```

The resource itself looked configured correctly, and was fine, but subsequent terraform plan showed that it would be replaced because this caused it to be tainted.

This is v0.0.4

**Comment by Kinzeng** [comment]
this should be addressed in #34

**Comment by guykoh** [comment]
@Kinzeng is this fixed now? Can we close this issue?
